GUI/MetaDraw/MetaDraw.xaml.cs

```diff
@@ -48,6 +48,8 @@ public partial class MetaDraw : Window
         private SolidColorBrush modificationAnnotationColor;
         private Regex illegalInFileName = new Regex(@"[\\/:*?""<>|]");
         private ObservableCollection<string> plotTypes;
```

Comment:
i feel like we are going to run into case sensitivity problems. is that an issue? if yes,  I think it can be dealt with

Reply:
the ".ToLower()" on line 129 (new code, line 127 in the old code) should take care of it